crm_department/__init__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```

Comment:
OK, if you don't want to do it, it's not a blocking point, but it's very quick to do it and we clean up a lot.